<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="schuifdeurKnop">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hallsensor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>s1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideur1Knop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>d1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideur2Knop">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>d2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHalsensor">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>halsensor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="schuifPassSlot1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; }</string>
    </property>
    <property name="placeholderText">
     <string>Enter password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiCodeSlot2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; }</string>
    </property>
    <property name="placeholderText">
     <string>Enter code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiCodeSlot1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; }</string>
    </property>
    <property name="placeholderText">
     <string>Enter code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="schuifCodeSlot1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; }</string>
    </property>
    <property name="placeholderText">
     <string>Enter code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiPassSlot1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; }</string>
    </property>
    <property name="placeholderText">
     <string>Enter password:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
